### PR TITLE
Product detail page

### PR DESCRIPTION
**Pull request by kaustubh-jsr**
Files to check:
1.ProductDetail.jsx
2.ProductDetail.css
3.ProductDetailContainer.jsx

### CODE REVIEW

**Review by manojasarsa — APPROVED** [review]
Code looks clean!

File: src/Pages/ProductDetail/ProductDetail.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from "react";
+import { Navigate, useNavigate, useParams } from "react-router-dom";
+import { getItemDetailApi } from "../../apis";
+import ProductDescription from "../../components/ProductDetail/ProductDescription";
+import ProductDetailContainer from "../../components/ProductDetail/ProductDetailContainer";
```
Comment (anasm11) [inline src/Pages/ProductDetail/ProductDetail.jsx:5]: You can make index file for components and import from there.

File: src/components/ProductDetail/ProductDetailContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import ReactImageMagnify from "react-image-magnify";
+import { useAuth } from "../../contexts/auth-context";
+import { useCart } from "../../contexts/cart-context";
+import { useWishlist } from "../../contexts/wishlist-context";
```
Comment (anasm11) [inline src/components/ProductDetail/ProductDetailContainer.jsx:6]: same here. index file can be made for contexts.

**Review by anasm11 — APPROVED** [review]
Suggested a change. Rest all looks good. You can merge.

**Pull request merged**